Userland/Libraries/LibAudio/GenericTypes.h

```diff
@@ -40,7 +40,7 @@ enum class ID3PictureType : u32 {
 // Note: This was first implemented for Flac but is compatible with ID3v2
 struct PictureData {
     ID3PictureType type {};
-    DeprecatedString mime_string {};
+    StringView mime_string {};
```

Comment:
This looks suspicious, replacing a `DeprecatedString` with a `StringView` is not an equivalent exchange. `DeprecatedString` owns its memory, whereas `StringView` is just a reference to some existing string, if you construct `StringView` to a temporary value you'll get a use-after-free.